src/test.js

```diff
@@ -9,7 +9,7 @@ describe('MenuButton', () => {
   })
 
   it('has not active class when selected not equals id', () => {
-    const component = renderer.create(
+    const tree = renderer.create(
```

Comment:
we need one more test for glowing I think

Reply:
done